packages/bits/demo/src/components/demo/tabgroup/tabgroup-docs/tabgroup-docs.example.component.html

```diff
@@ -229,7 +229,9 @@ <h2>
 <p>
     <nui-message type="warning" allowDismiss="true">
         Because this example is in the iframe you won't see the route changes. To watch routes change in browser's address bar please navigate
-        to <a href="./#/tabgroup" target="_blank">this exmaples page</a>. The link opens in a new tab.
+        to <a href="./#/tabgroup" target="_blank">this exmaples page</a>. The link opens in a new tab. <br>
+        Note that for keyboard navigation purpose you need to hardcode tabindex="-1" for nui-tab-heading to avoid issue with navigation.
+        I's because routerLink itself adds tabindex.
```

Comment:
```suggestion
        Note that for keyboard navigation purposes you need to hardcode tabindex="-1" for nui-tab-heading to avoid issues with navigation.
        It's because routerLink itself adds tabindex.
```

Reply:
thanks a lot.